tools/apps/src/bin/quiche-client.rs

```diff
@@ -139,6 +139,21 @@ fn main() {
     let mut conn =
         quiche::connect(connect_url.domain(), &scid, &mut config).unwrap();
 
+    // Only bother with qlog if the user specified it.
+    #[cfg(feature = "qlog")]
```

Comment:
FTR, https://github.com/rust-lang/rust/pull/69201 will be in Rust 1.43, so we can probably simplify this a bit in the future.